specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/server.should.timeout.if.client.does.not.respond.to.extended.ping/request.rpt

```diff
@@ -18,20 +18,24 @@ read version "HTTP/1.1"
 read header "Upgrade" /(?i:websocket)/
 read header "Connection" /(?i:Upgrade)/
 read header "Sec-WebSocket-Accept" ${ws:handshakeHash(key)}
-# The 4 control bytes for ping-pong extension are 0x01010102:
-read header "Sec-WebSocket-Extensions" "x-kaazing-ping-pong; 01010102"
+read header "Sec-WebSocket-Extensions" /x-kaazing-ping-pong; *(?<controlBytes>[a-fA-F0-9]{8})/
 
 # Expect extended PING
-read [0x81 0x06 0x01 0x01 0x01 0x02 0x09 0x00]
+read [0x81 0x06]
+read ${pp:parseHexString(controlBytes)}
+read [0x01 0x01 0x01 0x02 0x09 0x00]
```

Comment:
You need to remove the control bites from this line (0x01 0x01 0x01 0x02), otherwise the test fails. Everything goes okay on my machine when I make this change.